src/operations_problems_templates.jl

```diff
@@ -123,10 +123,20 @@ ed_problem = EconomicDispatchProblem(system)
 - `devices::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
 - `branches::Dict{Symbol, DeviceModel}` : override default `DeviceModel` settings
 - `services::Dict{Symbol, ServiceModel}` : override default `ServiceModel` settings
+- Key word arguments supported by `OperationsProblem`
 """
 
 function EconomicDispatchProblem(system::PSY.System; kwargs...)
-    template = template_economic_dispatch(; kwargs...)
+    kwargs = Dict(kwargs)
+    template_kwargs = Dict()
+    for (kw, arg) in kwargs
+        if !(kw in OPERATIONS_ACCEPTED_KWARGS)
+            template_kwargs[kw] = kwargs[kw]
+            pop!(kwargs, kw)
```

Comment:
Similar to my other comment but `setdiff`.

`pop!` returns the value so you could combine lines 134 and 135.